packages/cascara/src/components/Table/TableBase.js

```diff
@@ -129,7 +164,24 @@ const TableBase = ({
     ? inferUniqueID(Object.keys(data[0]))
     : undefined;
 
-  // // FDS-142: new action props
+  // FDS-518: sort table records
+  const sortedData = useMemo(() => {
+    if (!sortState?.attribute || sortState?.order === SORT_ORDER.UNSORTED) {
```

Comment:
(Optional) `attribute` and `order` property names always exist in the state, you could drop the `?.` null safe operator

Reply:
This is a remanent of the time when we were building the reducer. It's ok to remove it now. Tanks for the catch!